### ISSUE TITLE
Update for ember 4

### ISSUE THREAD

**Issue by Timbuktu1982** [emberjs/ember-copy#18]
Hi there, 
is it planned to update this addon to ember 4?

**Comment by pjcarly** [comment]
Interested in this as well.

**Comment by neilthawani** [comment]
Any ember 4 updates in the works?

**Comment by acorncom** [comment]
@neilthawani no, from what I've seen no updates are planned here. Suggestions are to copy using lodash libraries or other common approaches
